include/sm64.h

```diff
@@ -15,11 +15,21 @@
 #include "macros.h"
 #include "segments.h"
 
-// Crash handler enhancement
-#ifdef CRASH_SCREEN_INCLUDED
-#define DEBUG_ASSERT(exp) do { if (!(exp)) _n64_assert(__FILE__, __LINE__, #exp, 1); } while (0)
+extern char *__n64Assert_Filename;
+extern u32   __n64Assert_LineNum;
+extern char *__n64Assert_Message;
+extern void __n64Assert(char *fileName, u32 lineNum, char *message);
+
+#define ASSERT(cond, message) do {\
```

Comment:
imo if this is a runtime assert that follows the C standard then it should be named `assert` in all lowercase

Reply:
on second thought that requires a bit of extra work (including meddling with n64_assert.s and the assert parser) so consider this comment optional